mmf/utils/visualize.py

```diff
@@ -45,3 +48,27 @@ def visualize_images(
 
     plt.axis("off")
     plt.imshow(grid.permute(1, 2, 0))
+
+
+def visualize_frcnn_features(image_path, features_path, objids, attrids):
```

Comment:
Add typings here as well

Reply:
Might be good to write a test for this method. One way to test is make sure the input and output image arrays are same.